react-shim.js

```diff
@@ -0,0 +1,2 @@
+import { jsx } from '@emotion/react';
+export { jsx };
```

Comment:
This is a bit quirky. Is this file only needed for the esbuild? If so, is there a way we can indicate this - maybe with colocation, maybe with naming, maybe with a comment, etc. so that it's not misunderstood

Reply:
this is doing the job of `@emotion/babel-reset-css-prop` (docs: https://emotion.sh/docs/@emotion/babel-preset-css-prop#usage), it does the job of injecting an import from `@emotion/react` for `jsx` which is needed to enable support for the `css` prop. Here's a good thread where it's discussed: https://github.com/evanw/esbuild/issues/832#issuecomment-781497513

I think a clarifying comment here is an excellent suggestion